platform/ios/resources/es.lproj/Localizable.strings

```diff
@@ -31,18 +34,33 @@
 /* Accessibility label */
 "INFO_A11Y_LABEL" = "Acerca de este mapa";
 
+/* User-friendly error description */
+"LOAD_MAP_FAILED_DESC" = "No se pudo cargar el mapa debido a un error desconocido.";
+
+/* User-friendly error description */
+"LOAD_STYLE_FAILED_DESC" = "No se pudo cargar el mapa debido a un error de carga en el estilo.";
+
 /* Accessibility label */
 "LOGO_A11Y_LABEL" = "Mapbox";
 
 /* Accessibility label */
 "MAP_A11Y_LABEL" = "Mapa";
 
 /* Map accessibility value */
-"MAP_A11Y_VALUE" = "Zoom %1$dx\nAnotaciones visibles: %2$ld";
+"MAP_A11Y_VALUE" = "Zoom x%1$d\nAnotaciones visibles: %2$ld";
+
+/* User-friendly error description */
+"PARSE_STYLE_FAILED_DESC" = "No se pudo cargar el mapa debido a que el estilo está dañado.";
 
 /* Action sheet title */
 "SDK_NAME" = "Mapbox iOS SDK";
 
+/* Developer-only SDK update notification; {latest version, in format x.x.x} */
+"SDK_UPDATE_AVAILABLE" = "Mapbox iOS SDK versión %@ ahora es disponible:";
```

Comment:
The phrase that improves the meaning would be: "Mapbox iOS SDK versión %@ está disponible:" or even more specific: "La versión %@ del Mapbox iOS SDK está disponible:"

Reply:
I removed `l` this reads better "La versión %@ de Mapbox iOS SDK está disponible:"

Reply:
Fixed in Transifex and ceec08674d95c09af4573ecd79fb50c59724ab7b.